var/spack/packages/lmdb/package.py

```diff
@@ -0,0 +1,39 @@
+import os
+from spack import *
+
+class Lmdb(Package):
+    """Read-only mirror of official repo on openldap.org. Issues and
+    pull requests here are ignored. Use OpenLDAP ITS for issues.
+    http://www.openldap.org/software/repo.html"""
+
+
+    homepage = "http://www.openldap.org/software/repo.html"
+    url      = "https://github.com/LMDB/lmdb/archive/LMDB_0.9.16.tar.gz"
+
+    version('0.9.16', '0de89730b8f3f5711c2b3a4ba517b648')
+
+    def install(self, spec, prefix):
+        os.chdir('libraries/liblmdb')
```

Comment:
@adammoody: you can use `cd('libraries/liblmdb')` here, but this is fine.  I can merge this.